homeassistant/components/nintendo_parental/strings.json

```diff
@@ -0,0 +1,38 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "description": "To obtain your access token, click [Nintendo Login]({link}) to sign in to your Nintendo account. Then, for the account you want to link, right-click on the red **Select this person** button and choose **Copy Link Address**.",
```

Comment:
Not everyone sets this up on desktop tho

Reply:
I'm not sure how best to address this - its stated in the docs that you need to remove the Nintendo Switch Parental Controls app first (and the reason why) https://github.com/home-assistant/home-assistant.io/blob/fdd531e60564e5e34ceba216955a40e6931d9ee3/source/_integrations/nintendo_parental.markdown

"open the context menu" sounds even more confusing, but you need that link to be able to proceed.